introduction.md

```diff
@@ -0,0 +1 @@
+Hello my name is Randall J Collins Jr and I am a programmer utilizing code school for learning.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.